### DISCUSSION TITLE
Playlist error

### DISCUSSION THREAD

**Discussion by Aussie-boii** [maxrave-dev/SimpMusic#447] [General]
Hello I have a issue when I try to delete a song of my playlist it gives me a error and wants me to restart app when I try it again it does the same issue![Screenshot_20240712-133347.png](https://github.com/user-attachments/assets/859fdf99-6f5b-4cbc-98eb-231adbc39f4c)

**Comment by Ctrl-sza18** [comment]
Same happens to me Maxrave can you pls fix this error

**Comment by maxrave-dev** [comment]
Do you try to sync this playlist to YouTube Music and remove local playlist then sync this again
